### PR TITLE
pyDKB: EOM/EOP markers

### PR DESCRIPTION
**Pull request by anastasiakaida**
- [x] Implement EOM/EOP shortcuts.
- [x] Implement custom markers from the console.
- [x] Set up stdin in a non-blocking mode.
- [x] Implement custom readline function.
- [x] Mention the shortcut table in the --mode parameter description or epilog.
- [x] Add KeyboardInterrupt handler.

### CODE REVIEW

File: Utils/Dataflow/pyDKB/dataflow/stage/AbstractStage.py [hunk 1]
```diff
@@ -54,6 +89,31 @@ def parse_args(self, args):
             raise ValueError(
                 "Parameter -m|--mode must be used with value: -m MODE.")
 
+        if self.ARGS.eom is None:
+            self.ARGS.eom = '\n'
+        elif self.ARGS.eom == '':
+            raise ValueError("(ERROR) Empty EOM marker specified.\n")
```
Comment (mgolosova) [inline Utils/Dataflow/pyDKB/dataflow/stage/AbstractStage.py]: Just a comment: `\n` in the end of the `Exception` message is not necessary:
```
>>> raise ValueError("My message.\n")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: My message.

>>> raise ValueError("My message.")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: My message.
>>>
```

**Comment by anastasiakaida** [comment]
@mgolosova 
Now I see that joining of 09b63ef, c6d0cc7, 8473c6f is a reasonable decision. But some days ago for me, it looked like a set of separated units which shouldn't be joined. So... I'll fix it. Thank you! 
_____
Some words about 1c1a2c3: As I understood, I should just rewrite this commit description, right?
______
I'll try to fix it all till this evening. Thank you!

**Comment by mgolosova** [comment]
@anastasiakaida,
:) Ok, thank you!

Yes, my note to 1c1a2c3 was about the log message.

**Comment by anastasiakaida** [comment]
@mgolosova 
Now it seems that everything is ok now. Please, check this PR again. :)

File: Utils/Dataflow/pyDKB/dataflow/stage/AbstractProcessorStage.py [hunk 2]
```diff
@@ -344,7 +341,7 @@ def output(self, message):
     def forward(self):
         """ Send EOPMessage in the streaming output mode. """
         if self.ARGS.dest == 's':
```
[the remark this answers is not in the document]
Comment (anastasiakaida) [reply Utils/Dataflow/pyDKB/dataflow/stage/AbstractProcessorStage.py]: @mgolosova 
Good question. I suppose, not at all. :)

**Comment by mgolosova** [comment]
5837dec -- I would say that "odd results" are quite predictable: if the EOM marker is empty, how can we say message 1 from message 2 in the stream ("msg1msg2abcdefg")?

60cd0ac -- a pair of typos in the commit log: "lof message" and "grogram logs".

Everything else looks fine now. Thank you for your hard work! :)

---
Concerning the comment about `if` statement in the `forward()` message: I think now, in this PR, it can be left as is; not sure that we can make some use of printing EOP into outut files... but what we got to think about is: when do we need EOP exactly? 
I mean -- EOP is required only by Kafka supervisor, right? And it is required to decide if it should send next message for processing. Maybe we should think about leaving STDIN and STDOUT as pure data streams, and create a new pair -- CMDIN и CMDOUT, to use as a command channel.
Will think about it soon :)

**Comment by mgolosova** [comment]
Please check the last line in the log message of 22bed2b :)

**Comment by anastasiakaida** [comment]
@mgolosova 
Fixed :)

**Pull request merged**